lib/setupmanagers/qemuhck/devices/amd-iommu.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "iommu",
+  "command_line": [
+    "-device amd-iommu,intremap=on,device-iotlb=on,xtsup=on"
```

Comment:
`CmdRun PID:59231: stderr: qemu-system-x86_64: -device amd-iommu,intremap=on,device-iotlb=on,xtsup=on: Property 'amd-iommu.xtsup' not found`

Reply:
It works well without `xtsup=on`

Reply:
This is a new flag that was added by the AMD team. Thanks for testing.